kumo-cli/src/test/java/com/kennycason/kumo/cli/KumoCliITest.java

```diff
@@ -1,24 +1,17 @@
 package com.kennycason.kumo.cli;
 
-import com.kennycason.kumo.IntegrationTest;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-
 /**
  * Created by kenny on 6/12/16.
  */
-@Category(IntegrationTest.class)
```

Comment:
I think these should be marked as @Ignore so that we at least know they are tests. Later I'll create a better profile/config for integration tests.